assets/questions/longestSub.js

```diff
@@ -0,0 +1,96 @@
+// Copyright 2017 The TIE Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Question data for the longest substring.
+ */
+
+globalData.questions['longestSub'] = {  // eslint-disable-line dot-notation
+  title: 'Longest Substring',
+  starterCode: {
+    python:
+`def longestSubstring(seq):
+    return ""
+`
+  },
+  auxiliaryCode: {
+    python:
+`class AuxiliaryCode(object):
```

Comment:
Spaces around = all through these methods.

Please make sure you're writing good Python in these, since we're not deploying a linter on them.